### ISSUE TITLE
Bitwig does not recognize midi controller

### ISSUE THREAD

**Issue by DaKingof** [NixOS/nixpkgs#364099]
After an update(not sure which) Bitwig no longer recognizes my Novation Launchpad MK3 Pro. I've tried using MIDI through my web browser, and it works perfectly fine. I see there was a new wrapper added and some other changes recently about when I noticed this broke so I suspect Bitwig can no longer see my system hardware USB devices. 

```
lsusb
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 001 Device 002: ID 1235:0123 Focusrite-Novation Launchpad Pro MK3
Bus 001 Device 003: ID 8087:0aa7 Intel Corp. Wireless-AC 3168 Bluetooth
Bus 001 Device 004: ID 046d:c077 Logitech, Inc. Mouse
Bus 001 Device 005: ID 04ca:004b Lite-On Technology Corp. Keyboard
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 002 Device 002: ID 174c:55aa ASMedia Technology Inc. ASM1051E SATA 6Gb/s bridge, ASM1053E SATA 6Gb/s bridge, ASM1153 SATA 3Gb/s bridge, ASM1153E SATA 6Gb/s bridge
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 003 Device 002: ID 1462:7c37 Micro Star International MYSTIC LIGHT 
Bus 003 Device 003: ID 05e3:0608 Genesys Logic, Inc. Hub
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 005 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 005 Device 002: ID 17cc:1978 Native Instruments Audio 8 DJ
Bus 006 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
```

 - system: `"x86_64-linux"`
 - host os: `Linux 6.11.0-rt7, NixOS, 25.05 (Warbler), 25.05.git.a73246e2eM`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.10`
 - channels(root): `"home-manager, musnix, nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

@bfortz @michalrus @mrVanDalo @polygon @chewblacka

**Comment by bfortz** [comment]
Fixed in #361845.

**Comment by DaKingof** [comment]
Thank you @bfortz 

This now works just fine for me(seemingly better than before even as my controller actually auto-detects now when it never did before).
